### PR TITLE
Adding stub project for the API Design Activity for SDK Training

### PR DESCRIPTION
**Pull request by annelo-msft**
See related PR: https://github.com/Azure/azure-sdk-pr/pull/360

### CODE REVIEW

File: sdk/api-learn/Azure.Learn.Computation/src/Azure.Learn.Computation.csproj [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>API Design Learning Project</Description>
+    <AssemblyTitle></AssemblyTitle>
+    <Version>0.0.1-preview.1</Version>
```
Comment (christothes) [inline sdk/api-learn/Azure.Learn.Computation/src/Azure.Learn.Computation.csproj:5]: Should we start snapping to the new `beta` moniker`?

File: sdk/api-learn/Azure.Learn.Computation/tests/ComputationClientTestEnvironment.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+
+namespace Azure.Learn.Computation.Samples
+{
+    // To learn more about TestEnvironment classes, please see: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core.TestFramework/README.md
+    public class ComputationClientTestEnvironment : TestEnvironment
+    {
+        public ComputationClientTestEnvironment() : base("api-learn")
```
Comment (christothes) [inline sdk/api-learn/Azure.Learn.Computation/tests/ComputationClientTestEnvironment.cs:11]: should the string we pass to the base ctor match to the service name?

**Review by christothes — APPROVED** [review]
👍

File: sdk/api-learn/Azure.Learn.Computation/tests/ComputationClientTestEnvironment.cs (hunk 2 above)
Comment (annelo-msft) [reply sdk/api-learn/Azure.Learn.Computation/tests/ComputationClientTestEnvironment.cs:11]: https://github.com/Azure/azure-sdk-pr/issues/403

**Comment by jsquire** [comment]
Hi @annelo-msft.   Can you help me understand if this work is still active or we should close out the PR?

**Pull request merged**